src/sparseml/pytorch/utils/sparsification.py

```diff
@@ -200,7 +200,7 @@ class GradSampler:
     def __init__(
         self,
         data_loader: Union[Iterator[Tuple[List[Any], Dict[str, Any], Any]], Callable],
-        loss_fn: Callable[[Any], Any],
+        loss_fn: Callable[[Any, Any], Any],
```

Comment:
There are loss functions that take 3 arguments: https://pytorch.org/docs/stable/generated/torch.nn.TripletMarginLoss.html#torch.nn.TripletMarginLoss

Make sure that only loss functions with 2 arguments are passed to `loss_fn`

Reply:
Never mind, function is only used in one place, lgtm